docs/index.md

```diff
@@ -1,12 +1,12 @@
 *ehrQL* (rhymes with *circle*), the **Electronic Health Records Query Language**,
-is a query language and command line interface designed for the specific application domain of EHR data.
+is a query language and command line interface that was designed for the specific application domain of EHR data.
```

Comment:
Was the original really unclear? Obviously, excessively elliptical constructions can be problematic; but to me that modified sentence is clunkier but no easier to parse.

Reply:
I've been reading _Developing Quality Technical Information_ and also thinking more about teaching/learning, which includes thinking more about people who don't speak English as their first language. I don't think the original was unclear; I was, however, persuaded that the revision was clearer when I saw a similar example translated into French and German, and how the "that was" made the translations more accurate.

Reply:
Right yes, this in an interesting point. I wonder if we might end up needing to explicitly distinguish different sorts of writing, with different sorts of goal and audience, within the Bennett Institute. There's definitely a place for maximising non-native readability (of which reference docs would be the obvious example) but I wouldn't  want to eviscerate all personality from our writing more generally.